admin/README.md

```diff
@@ -1,5 +1,28 @@
 # c2cgeoportal_admin
 
+## Run with docker
+
+To build:
+```
+make docker-build
+```
+
+To build & run the application:
+```
+make docker-serve
+```
+
+Open http://localhost:8888/
+
+The following views are also provided by `c2cwsgiutils`:
+ * http://localhost:8888/versions.json
+ * http://localhost:8888/stats.json
+ * http://localhost:8888/health_check?max_level=3
+
+
+## Run without docker
+
+
```

Comment:
Is running without docker unsupported? If no then I think we should keep the lines in gitignore and in the find command.